src/styles/_sizes.media.scss

```diff
@@ -52,3 +88,12 @@ $preview-medium-width: 1000px;
         @content;
     }
 }
+
+/* stylelint-disable */
+:export {
```

Comment:
I'm not sure I've seen this SCSS feature before. Can you explain more about it? Is it supported in all the browsers and build systems we use?

Reply:
Actually, `:export` is not an SCSS feature. It is part of the `css-loader` module. CSS Loader implements the Interoperable CSS standard that defines `:export` and `:import` behaviors: 

https://webpack.js.org/loaders/css-loader/#separating-interoperable-css-only-and-css-module-features

Reply:
I like this idea, but I'm concerned it could cause issues for platform developers who use the BUIE NPM package. Can we use the `variables.ts` generator script for these new variables, instead?

Reply:
Understood. Will make use of the `variables.ts` generator script.